com.unity.robotics.urdf-importer/Runtime/Extensions/UrdfGeometryCollision.cs

```diff
@@ -139,7 +139,7 @@ public static void CreateMatchingMeshCollision(Transform parent, Transform visua
             collisionObject.transform.SetParentAndAlign(parent);
         }
 
-        private static void ConvertMeshToColliders(GameObject gameObject, string location = null, bool setConvex = true)
+        private static void ConvertMeshToColliders(GameObject gameObject, string location = null, bool setConvex = true, bool cylinder = false)
```

Comment:
It is better to create a new separate function for creating colliders for standard geometry (Cylinder, as well as Box and Sphere) rather than making the ConvertMeshToColliders a multi-purpose function.

Reply:
I thought the that the source of the issue was that a custom mesh needs to be generated for cylinders since there isn't a [standard "cylinder collider,"](https://docs.unity3d.com/Manual/PrimitiveObjects.html) which is why boxes and spheres were able to be saved as prefabs as expected. Turtlebot works for this purpose, since it is already adding a `BoxCollider` (and spheres also already use `SphereCollider`), so the prefabs were still functioning well for me in that case

Reply:
You are absolutely right. I didn't test properly and the colliders are handled properly for the case of box and sphere.
Now I wonder if it is still would be possible to make it a cleaner code by resolve the location and the mesh filename, before calling `ConvertMeshToColliders()`.